.pre-commit-config.yaml

```diff
@@ -0,0 +1,11 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.3.0
+    hooks:
```

Comment:
When I run `pre-commit run --all-files` , all hooks but yaml ones fail.

end-of-file-fixer and trailing-whitespace changes all our binary files, which breaks the tests.
In addition trailing-whitespace hook changes `applications/test/cath.output` contents, which also breaks the test.


How do we want to deal with these?

How often would we be adding/changing files like that? I think it won't be a rare occasion. What is the plan then? Disable hooks every time manually? Sounds painful.

Should we add test/output files into pre-commit exclude lists instead?